src/key/ledger/app.ts

```
@@ -0,0 +1,231 @@
+
+  async getVersion(): Promise<string> {
+    const { version } = await this.getAppConfiguration()
+    return version
+  }
+
+  async getAppConfiguration(): Promise<any> {
+    return this.app.getAppConfiguration()
+  }
+
+  setLoadConfig(config: LoadConfig): void {
+    return this.app.setLoadConfig(config)
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)